### DISCUSSION TITLE
Highlight original handle when using an alias

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15597] [Ideas] [answered]
When using a custom field handle (I love this feature by the way), it would be great if the original handle was highlighted as well. We've found it somewhat difficult at times to track down or simple see what the underlying handle is. Currently, you have to open the field settings and temporarily delete the custom handle to see what the original is.

I slapped together this UI idea, but definitely could be refined or more thought out.

![CleanShot 2024-08-26 at 12 45 24@2x](https://github.com/user-attachments/assets/2d84ee29-9e25-4407-a228-ee236d43c713)

![CleanShot 2024-08-26 at 12 42 19@2x](https://github.com/user-attachments/assets/7083e7b9-55c8-46e3-92e5-e73eeab3be97)

**Comment by brandonkelly** [comment, marked as answer]
Don’t really like the idea of showing two handles (or two field names) within the field layout element selector, but I just made it easier to see the original name/handle by adding a chip to the top of the settings slideout with info from the global field, for Craft 5.4 (#15619).

**Reply by ryanleichty** [reply]
Awesome, that's a nice upgrade! 🔥

I still think having some visual indication within the field layout would be beneficial too. Given that you already have a pattern established with showing an icon for other modified settings, I think it would work well for this too.

![CleanShot 2024-08-29 at 08 51 12@2x](https://github.com/user-attachments/assets/66eafc89-9393-4bd0-b3f8-868f12e01a64)

**Reply by ryanleichty** [reply]
Should I submit this as a new idea?

**Reply by brandonkelly** [reply]
@ryanleichty Good idea, just made that change as well: (149225250c61ad4b71f9fbfe1b1db5561833e928)

<img src="https://github.com/user-attachments/assets/35706974-c352-471c-8f7a-b9233da6076c" width="251" height="80" alt="A custom field selector with a pencil icon, indicating that its attributes have been overridden">

The pencil icon will appear regardless of which attributes were overridden, and the specific attributes will be listed in its tooltip.

**Reply by ryanleichty** [reply]
Nice, the pencil in the green looks even better. Thanks so much!

**Reply by brandonkelly** [reply]
Craft 5.4.0 is out with both of those changes 🎉
